### DISCUSSION TITLE
Question on how I should handle releases

### DISCUSSION THREAD

**Discussion by DylanDevelops** [community/community#31899] [Other Feature Feedback, Questions, & Ideas] [answered]
So I am a game developer and for the first time ever I'm using GitHub to backup my files for making my games. I had a question. When I release a build I release it. After that, I make lots of changes and when I feel like it is ready for a new release I release a new build. I was wondering what the best way of handling this type of thing on GitHub might be. Should I create a new branch that I create all those new changes on and then when I am ready, I push everything from that branch to main? Is that how people handle releases with software and games? I'm kind of new to this so I wanted to hear what you guys had to say. Thanks in advance!

**Comment by radar-blip** [comment, marked as answer]
Sounds to me like you're on the right track, create a branch for testing purposes, and when everything is to your liking, merge it into the main branch. Then of course, create a new release.
Here is a quick read [about branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches).

**Reply by DylanDevelops** [reply]
Thank you!

**Reply by radar-blip** [reply]
You're welcome. Good luck in your gaming adventures!
